charts/elasticsearch/templates/nginx/nginx-es-ingress-networkpolicy.yaml

```diff
@@ -28,6 +28,7 @@ spec:
         matchLabels:
           tier: airflow
           component: webserver
+    {{- if .Values.global.loggingSidecar.enabled }}
```

Comment:
@rishkarajgi what was the answer to your question?

Reply:
i have answered the question asked by @rishkarajgi , he got clarified

Reply:
@rishkarajgi @danielhoherd what makes you think this is being restricted to Triggerer only?

Reply:
we have already added network labels for all airflow components.In this change we are restricting based on global.sidecarLogging  so that extra network labels will be allowed based on that

Reply:
@vishwas-astro @danielhoherd We are adding these extra network labels since we have moved sidecar config to globals